third_party/blink/renderer/platform/media/web_media_player_impl.cc

```diff
@@ -2805,6 +2805,9 @@ std::unique_ptr<media::Renderer> WebMediaPlayerImpl::CreateRenderer(
   // TODO(b/375278384): Select the StarboardRenderer properly instead of
   //                    hard coding.
 
+  // StarboardRenderer always uses full screen with overlay video mode.
+  overlay_info_.is_fullscreen = true;
```

Comment:
Shall we guard the code with `IS_COBALT` and `USE_STARBOARD_MEDIA`?

Reply:
It is gated already (with StarboardRenderer below).